src/components/ChecAlert.vue

```diff
@@ -141,6 +141,16 @@ $alert-inline-colors: (
       @apply inline h-4 w-4 relative;
       top: -1px;
     }
+
+    a {
+      @apply text-white font-bold;
+
+      &:hover,
+      &:focus,
+      &:active {
+        @apply text-white font-bold;
```

Comment:
Do you want to have something to visually indicate that it's being hovered, i.e. an underline?

Reply:
Yea I can add that in.

Reply:
@robbieaverill solved in #478.